src/main/resources/footerScripts.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
```

Comment:
You should place these fragments in `common.html`.

Reply:
fragments added to common.html